deps/eventlog/src/ccel/event_tag.rs

```diff
@@ -29,8 +37,51 @@ impl EventDataParser for EvEventTagParser {
             .get(index..index + data_size as usize)
             .ok_or_else(|| anyhow!("Failed to read description"))?;
 
-        let event_desc = String::from_utf8(event_data.to_vec())?.replace('\0', "");
-        Ok(EventDetails::from_string(event_desc))
+        let event_desc = String::from_utf8_lossy(event_data);
+        let mut details = EventDetails::from_string(event_desc.replace('\0', "").to_string());
+
+        if event_id == AAEL_TAGGED_EVENT_ID {
```

Comment:
Can you add CCEL binary example in test_data and include it in `test_query_digest` test ?